cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -53,11 +53,12 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   var _isExpanded = false;
   var _isAnimating = false;
 
-  var _expandTransition;
-  var _collapseTransition;
-  var _expandTransitionMethod;
+  var _expandTransition = null;
+  var _expandTransitionMethod = null;
   var _expandTransitionMethodArgs = [];
-  var _collapseTransitionMethod;
+
+  var _collapseTransition = null;
```

Comment:
Hmm, undefined vars actually do get set to the global singleton https://jsfiddle.net/xLc6fkze/

So the original problem I ran into was our linter prevents use of undefined, because I presume `_collapseTransition = undefined` would be setting a value (albeit undefined) to a variable that should not have a value. An undefined value on a variable should imply it's never been initialized before with a value. Since the transition can be added and removed, the default value needs to be reset. So I'm explicitly giving it a default value, similar to how `_collapseTransitionMethodArgs` has a default of an empty array. If I didn't initialize it as `null`, it could return three possible values: `null`, `undefined`, or an actual transition object, so setting it to null removes the ambiguity of having it return either undefined or null. I imagine code like dom querying is doing something similar behind the scenes.